### ISSUE TITLE
Option to automatically include peculiar offsets

### ISSUE THREAD

**Issue by varenius** [whi-llc/fmout#1]
For correlation, the fitted linear clock model + peculiar offsets have to be included in the VEX file. fmout.py gives the former, but not the latter. Maybe this is a feature, but to me it would make sense to include an option for fmout.py to spit out clock values with the respective station peculiar offset already included in the output time. Could this be an option? (My homegrown script, constructed before I knew about fmout, does this and it saves me some time and reduces the risk of applying offsets with wrong sign etc.)

**Comment by whimwich** [comment]
Of course. And I suppose it could also optionally output the information in the format of VEX `clock_early` statements.  :)
That could be a good way to make a preliminary clock model for correlation.

In the short-term, I would propose importing a table of peculiar offsets from _adjust_. It is not clear to me yet if we should include the table in the repo or have the user provide it. Longer term, we could have `fmout` by default include the table from the latest local `pull` of _adjust_. 

Once the clock model has been refined by a test correlation. It should be run through `adjust` to trim the overall peculiar offset, or at least check that it is good enough. Some input and output options in `adjust` could help streamiline that as well.
